contribs/gmf/src/directives/timeslider.js

```diff
@@ -0,0 +1,170 @@
+goog.provide('gmf.TimeSliderDirective');
+goog.provide('gmf.TimeSliderController');
+
+goog.require('gmf');
+
+gmf.module.value('gmfTimeSliderTemplateUrl',
+    /**
+     * @param {angular.JQLite} element Element.
+     * @param {angular.Attributes} attrs Attributes.
+     * @return {string} Template URL.
+     */
+    function(element, attrs) {
+      var templateUrl = attrs['gmfTimeSliderTemplateUrl'];
+      return templateUrl !== undefined ? templateUrl :
+          gmf.baseTemplateUrl + '/timeslider.html';
+    });
+
+
+/**
+ * Provide a directive to select a single date or a range of dates with a slider
```

Comment:
No, I think there is no need for this directive to change the HTML implementation. I'll remove it